app/models/warehouse_report/outcomes/th_report.rb

```diff
@@ -0,0 +1,50 @@
+###
+# Copyright 2016 - 2020 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# dev projects -- with affiliation: 61; single RRH: 44
+
+class WarehouseReport::Outcomes::ThReport < WarehouseReport::Outcomes::Base
+  include ArelHelper
+
+  def default_support_columns
+    {
+      service_project: _('Housing Search'),
+      search_start: _('Search Start'),
+      search_end: _('Search End'),
+      residential_project: _('Stabilization Project'),
+      housed_date: _('Date Housed'),
+      housing_exit: _('Housing Exit'),
+    }
+  end
+
+  def columns_for_percent_exiting_pre_placement
+    {
+      service_project: _('Housing Search'),
+      search_start: _('Search Start'),
+      search_end: _('Search End'),
+      housed_date: _('Date Housed'),
+    }
+  end
+
+  def columns_for_percent_in_stabilization
+    {
+      service_project: _('Housing Search'),
+      search_start: _('Search Start'),
+      search_end: _('Search End'),
+      residential_project: _('Stabilization Project'),
+      housed_date: _('Date Housed'),
+      housing_exit: _('Housing Exit'),
+    }
+  end
```

Comment:
Same here as with ES.  Maybe these should exist in `WarehouseReport::Outcomes::Base`?